docs/tutorials/data_management.md

```diff
@@ -139,8 +139,19 @@ The `mod` format will generate a Rust module that defines a data provider. This
 
 Let's give it a try:
 
+For the mod generation to work, you need an actual application that compiles. You can keep the existing app if you have been following this tutorial from the start.
```

Comment:
This is true for all generation types, there's nothing different about mod (after the bug fix). The expected way to follow a tutorial is to do all steps, so adding this section overcomplicates things.

Reply:
OK ! Removed !